sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1223,16 +1223,19 @@ class HiveDDLSuite
   }
 
   test("CREATE TABLE LIKE a temporary view") {
-    // CREATE TABLE LIKE a temporary view.
-    withCreateTableLikeTempView(location = None)
+    Seq(None, Some("parquet"), Some("orc")) foreach { provider =>
```

Comment:
How about just using `hiveFormats` here?

Reply:
Add hive or just hive?

Reply:
Just like `hiveFormats.foreach { provider =>`?

Reply:
Oh. Check the description please. This PR is to address the first item.
STORED AS hiveFormat will be open when this done.
https://github.com/apache/spark/pull/26097#issue-327424759